geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/LuceneServiceImpl.java

```diff
@@ -133,6 +135,22 @@ public void beforeRegionDestroyed(Region region) {
     }
   }
 
+  @Override
+  public void cleanupFailedInitialization(Region region) {
+    List<LuceneIndexCreationProfile> definedIndexes = getDefinedIndexes(region.getFullPath());
+    for (LuceneIndexCreationProfile definedIndex : definedIndexes) {
+      // Get the AsyncEventQueue
+      String aeqId = LuceneServiceImpl.getUniqueIndexName(definedIndex.getIndexName(),
+          definedIndex.getRegionPath());
+      AsyncEventQueueImpl aeq = (AsyncEventQueueImpl) cache.getAsyncEventQueue(aeqId);
+      // Stop and remove the AsyncEventQueue if it exists
+      if (aeq != null) {
+        aeq.stop();
+        this.cache.removeAsyncEventQueue(aeq);
```

Comment:
Does it need to send msg to other members to remove the AEQ?

Reply:
No, it doesn't need to do anything with remote members. This member is the one with the bad region definition, so its just this one that has to remove the AEQ. And it shouldn't remove anything else (like the index definition or RegionListener) in case the region is created correctly later.